pkg/services/authn/authnimpl/service.go

```diff
@@ -537,14 +537,29 @@ func parseNamespace(path string) string {
 	return parts[0]
 }
 
-// name of query string used to target specific org for request
+// name of the query string used to target specific org for request
 const orgIDTargetQuery = "targetOrgId"
```

Comment:
Hey @mgyongyosi I was testing the original PR in my environment and I stumbled upon this targetOrgId query param and this PR https://github.com/grafana/grafana/pull/60528/files.

I tried to set the targetOrgId to match the orgId query param and this actually solved my issue.

The PR I linked is talking about some authentication requests and I think the JWT requests might have been in them :D 

Anyway, I have done some shallow research and I only find a reference to the ?targetOrgId query param in one typescript file so I'm wondering it that is not some legacy that should be renamed to orgId instead of targetOrgId everywhere?

What do you think about this?